docs/kafka/message-browsing-kafka/README.adoc

```diff
@@ -0,0 +1,158 @@
+////
+START GENERATED ATTRIBUTES
+WARNING: This content is generated by running npm --prefix .build run generate:attributes
+////
+
+//OpenShift Application Services
+:org-name: Application Services
+:product-long-rhoas: OpenShift Application Services
+:community:
+:imagesdir: ./images
+:property-file-name: app-services.properties
+:samples-git-repo: https://github.com/redhat-developer/app-services-guides
+:base-url: https://github.com/redhat-developer/app-services-guides/tree/main/docs/
+
+//OpenShift Application Services CLI
+:rhoas-cli-base-url: https://github.com/redhat-developer/app-services-cli/tree/main/docs/
+:rhoas-cli-ref-url: commands
+:rhoas-cli-installation-url: rhoas/rhoas-cli-installation/README.adoc
+
+//OpenShift Streams for Apache Kafka
+:product-long-kafka: OpenShift Streams for Apache Kafka
+:product-kafka: Streams for Apache Kafka
+:product-version-kafka: 1
+:service-url-kafka: https://console.redhat.com/application-services/streams/
+:getting-started-url-kafka: kafka/getting-started-kafka/README.adoc
+:kafka-bin-scripts-url-kafka: kafka/kafka-bin-scripts-kafka/README.adoc
+:kafkacat-url-kafka: kafka/kcat-kafka/README.adoc
+:quarkus-url-kafka: kafka/quarkus-kafka/README.adoc
+:nodejs-url-kafka: kafka/nodejs-kafka/README.adoc
+:rhoas-cli-getting-started-url-kafka: kafka/rhoas-cli-getting-started-kafka/README.adoc
+:topic-config-url-kafka: kafka/topic-configuration-kafka/README.adoc
+:consumer-config-url-kafka: kafka/consumer-configuration-kafka/README.adoc
+:access-mgmt-url-kafka: kafka/access-mgmt-kafka/README.adoc
+:metrics-monitoring-url-kafka: kafka/metrics-monitoring-kafka/README.adoc
+:service-binding-url-kafka: kafka/service-binding-kafka/README.adoc
+
+//OpenShift Service Registry
+:product-long-registry: OpenShift Service Registry
+:product-registry: Service Registry
+:registry: Service Registry
+:product-version-registry: 1
+:service-url-registry: https://console.redhat.com/application-services/service-registry/
+:getting-started-url-registry: registry/getting-started-registry/README.adoc
+:quarkus-url-registry: registry/quarkus-registry/README.adoc
+:rhoas-cli-getting-started-url-registry: registry/rhoas-cli-getting-started-registry/README.adoc
+:access-mgmt-url-registry: registry/access-mgmt-registry/README.adoc
+:content-rules-registry: https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry/1/guide/9b0fdf14-f0d6-4d7f-8637-3ac9e2069817[Supported Service Registry content and rules]
+:service-binding-url-registry: registry/service-binding-registry/README.adoc
+
+//OpenShift Connectors
+:product-long-connectors: OpenShift Connectors
+:service-url-connectors: https://console.redhat.com/application-services/connectors
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-browsing-messages"]
+= Browsing messages in the {product-long-kafka} web console
```

Comment:
I'm sure you've already thought through this every which way, but I'm wondering if this would be better as just a procedure in the "Configuring topics" story rather than it's own dedicated story. That doc is for viewing and editing topic properties and also mapping topics with schemas, so I think it's a reasonable fit to move the key points from this assembly intro down into the proc, and then move the proc to that story. Wdyt?

Reply:
@sterobin Yeah, I did consider doing exactly that. However, in this case, users are not actually doing any configuration - they are just browsing and filtering messages. I didn't see it as a _great_ fit in that guide and with a risk of being missed by plenty of users. If you don't have very strong feelings about it, I'd prefer to let it stand alone for now. Would you be OK with that?

Reply:
Absolutely. Was just a thought that I wanted to raise. Thanks.